pkg/signature/verifier.go

```diff
@@ -43,8 +44,8 @@ func LoadVerifier(publicKey crypto.PublicKey, hashFunc crypto.Hash) (Verifier, e
 		return LoadRSAPKCS1v15Verifier(pk, hashFunc)
 	case *ecdsa.PublicKey:
 		return LoadECDSAVerifier(pk, hashFunc)
-	case *ed25519.PublicKey:
-		return LoadED25519Verifier(*pk)
+	case ed25519.PublicKey:
```

Comment:
I guess you meant to remove the pointer here (line 47)?

Reply:
yes, this was intentional. `rsa` and `ecdsa` represent their public key in structs, but `ed25519` just has a `[]byte` representation so they're handled differently.